scripts/globals/maws.lua

```diff
@@ -0,0 +1,161 @@
+    [ZN.MERIPHATAUD_MOUNTAINS]   = {bit = 5, group = 2, cs1 = 905, cs2 = nil, cs3 = nil, dest = { 595.000, -32.000,  279.300,  93,  97}},
+    [ZN.EAST_RONFAURE]           = {bit = 6, group = 2, cs1 = 904, cs2 = nil, cs3 = nil, dest = { 322.057, -60.059,  503.712,  64,  81}},
+    [ZN.NORTH_GUSTABERG]         = {bit = 7, group = 2, cs1 = 903, cs2 = nil, cs3 = nil, dest = { 469.697,  -0.050,  478.949,   0,  88}},
+    [ZN.WEST_SARUTABARUTA]       = {bit = 8, group = 2, cs1 = 904, cs2 = nil, cs3 = nil, dest = {   2.628,  -0.150, -166.562,  32,  95}},
+    [ZN.BATALLIA_DOWNS_S]        = {bit = 0, group = 3, cs1 = 701, cs2 = 100, cs3 = 101, dest = { -51.486,   0.371,  436.972, 128, 105}},
+    [ZN.ROLANBERRY_FIELDS_S]     = {bit = 1, group = 3, cs1 = 701, cs2 = 101, cs3 = 102, dest = {-196.500,   7.999,  361.192, 225, 110}},
+    [ZN.SAUROMUGUE_CHAMPAIGN_S]  = {bit = 2, group = 3, cs1 = 701, cs2 = 101, cs3 = 102, dest = { 366.858,   8.545, -228.861,  95, 120}},
+    [ZN.JUGNER_FOREST_S]         = {bit = 3, group = 4, cs1 = 101, cs2 = 102, cs3 = nil, dest = {-116.093,  -8.005, -520.041,   0, 104}},
+    [ZN.PASHHOW_MARSHLANDS_S]    = {bit = 4, group = 4, cs1 = 100, cs2 = 101, cs3 = nil, dest = { 415.945,  24.659,   25.611, 101, 109}},
+    [ZN.MERIPHATAUD_MOUNTAINS_S] = {bit = 5, group = 4, cs1 = 102, cs2 = 103, cs3 = nil, dest = { 595.000, -32.000,  279.300,  93, 119}},
+    [ZN.EAST_RONFAURE_S]         = {bit = 6, group = 4, cs1 = 100, cs2 = 101, cs3 = nil, dest = { 322.057, -60.059,  503.712,  64, 101}},
+    [ZN.NORTH_GUSTABERG_S]       = {bit = 7, group = 4, cs1 = 100, cs2 = 101, cs3 = nil, dest = { 469.697,  -0.050,  478.949,   0, 106}},
+    [ZN.WEST_SARUTABARUTA_S]     = {bit = 8, group = 4, cs1 = 100, cs2 = 101, cs3 = nil, dest = {   2.628,  -0.150, -166.562,  32, 115}},
+}
+
+local function meetsMission2Reqs(player)
+
+    if not player:getCurrentMission(WOTG) == dsp.mission.id.wotg.BACK_TO_THE_BEGINNING then
+        return false
+    end
+
+    local Q  = dsp.quest.id.crystalWar    
+    local Q1 = player:getQuestStatus(CRYSTAL_WAR, Q.CLAWS_OF_THE_GRIFFON) == QUEST_COMPLETED
+    local Q2 = player:getQuestStatus(CRYSTAL_WAR, Q.THE_TIGRESS_STRIKES)  == QUEST_COMPLETED
+    local Q3 = player:getQuestStatus(CRYSTAL_WAR, Q.FIRES_OF_DISCONTENT)  == QUEST_COMPLETED
+
+    return Q1 or Q2 or Q3
+
+end
+
+local function endMission2(player)
+
+    player:completeMission(WOTG, dsp.mission.id.wotg.BACK_TO_THE_BEGINNING);
+    player:addMission(WOTG, dsp.mission.id.wotg.CAIT_SITH);
+    player:addTitle(dsp.title.CAIT_SITHS_ASSISTANT);
+
+end
+
+local function endMission1(player)
+
+    local ID = require("scripts/zones/"..player:getZoneName().."/IDs")
+    player:messageSpecial(ID.text.KEYITEM_OBTAINED, dsp.ki.PURE_WHITE_FEATHER)
+    player:addKeyItem(dsp.ki.PURE_WHITE_FEATHER)
+    player:completeMission(WOTG,dsp.mission.id.wotg.CAVERNOUS_MAWS)
+    player:addMission(WOTG,dsp.mission.id.wotg.BACK_TO_THE_BEGINNING)
+
+end
```

Comment:
You can cut out these functions because you no longer need them to save space in your onEventFinish if checks down below